proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -53,6 +54,19 @@ message OrganizationUser {
   OrganizationRoleSource organization_role_source = 4;
 }
 
+message UserSelectedPlugin {
+  // owner of the plugin
+  string owner = 1;
+  // name of the plugin
+  string name = 2;
+}
+
+message UserPluginPreference {
+  string user_id = 1;
+  PluginLanguage selected_language = 2;
```

Comment:
Asymmetry between `UserSelectedPlugin` having `Selected` in it vs. this not, likely could be explained by understanding what API is and is for via docs